python/src/aiconfig/eval/metrics.py

```diff
@@ -1,11 +1,16 @@
+import json
 from functools import total_ordering
-from typing import Any, Generic
+from typing import Any, Generic, Type
 
+import lastmile_utils.lib.core.api as cu
 import nltk
 import pandas as pd
-from aiconfig.eval.common import CustomMetricValue, EvaluationFunction, EvaluationMetricMetadata, T_OutputDatum
+from aiconfig.eval import common
+from aiconfig.eval.common import CustomMetricValue, EvaluationFunction, EvaluationMetricMetadata, T_BaseModel, T_OutputDatum, TextRatingsData
```

Comment:
Is the first import needed (i.e. second one is redundant right now)?

Reply:
Yeah, it's inconsistent (see above), and I don't know if vscode can fix this. I'll try and get cli-mate to do it as a follow-up.